CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change log
 
-## master (unreleased)
+## main (unreleased)
+
+* [#58](https://github.com/Shopify/deprecation_toolkit/pull/58): Drop support for Ruby < 2.6 & Active Support < 5.2. (@sambostock)
+* [#58](https://github.com/Shopify/deprecation_toolkit/pull/58): `require 'active_support'` earlier to ensure compatibility with v7. (@sambostock)
```

Comment:
I've only documented the user facing changes. Excluded are:

- the changes to CI infrastructure
- the changes to development dependency versions and config